.github/workflows/emscripten-experimental.yml

```diff
@@ -0,0 +1,88 @@
+name: 'Test Build (Emscripten Experimental)'
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["Test Builds"]
+    branches: [master]
+    types:
+      - completed
+      - requested
+  schedule:
```

Comment:
If we haven't pushed to `master` in a while, `schedule` will run once a week on Friday so the `emsdk` team can see if something broke in the past week.